packages/arb-node-core/challenge/challenger.go

```diff
@@ -109,8 +70,8 @@ type SimpleChallengerImpl interface {
 type ChallengerImpl interface {
```

Comment:
Haven't looked all the way through this code, but I think dropping this interface probably makes sense assuming there's only a single implementation. It made much more sense before the other two challenge types were dropped